### DISCUSSION TITLE
Proposal to Honor Deceased GitHub Community Members

### DISCUSSION THREAD

**Discussion by 4i8** [community/community#121425] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello GitHub Team,

I recently learned about the passing of [Russel Winder](https://github.com/russel), a revered member of the developer community, on January 23, 2021. His passing is deeply saddening and it has prompted an idea to honor his memory and the memory of others who have passed away.

I suggest that GitHub could commemorate its deceased members by marking their profiles with a respectful note, such as "In Memoriam." This would not only honor their contributions but also serve as a poignant reminder of their impact within the GitHub community.

Adding such a feature would emphasize the enduring value of a developer's contributions and ensure that their legacies are acknowledged and remembered within the community.

Thank you for considering this proposal. I look forward to the community's thoughts on this tribute.

Best, 
Arth

**Comment by charlieforward9** [comment]
This is very interesting, honorable idea. 

I would add that - when the commemoration is added - the contribution graph accounts for the fact that no further contributions can be made, changing the visualization from tabs of blank dates.

**Reply by 4i8** [reply]
Thanks! That’s a solid point about the contribution graph, It definitely adds a meaningful touch to honoring their memory.

**Comment by ebndev** [comment]
Hi @4i8,

It's touching to hear that you are thinking of developers in the community and want to honor them.

Please note: this community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a product leader, but I did want to acknowledge your post!

**Reply by 4i8** [reply]
Hi @ebndev ,

Thank you for your kind words and guidance. I had already reached out to GitHub support before posting here, and they directed me to community/discussions. I’ll be contacting them again to follow up. Thanks again for your help!
